bodhi/client/__init__.py

```diff
@@ -236,6 +236,8 @@ def edit(user, password, url, **kwargs):
               help='Query only critical path packages')
 @click.option('--cves', help='Query by CVE id')
 @click.option('--packages', help='Query by package name(s)')
+@click.option('--content-type', help='Query updates based on content type',
+              type=click.Choice(['rpm', 'module']))  # And someday, container.
```

Comment:
I just don't know, but am open to opinions.

By defaulting to `None`, a query for `--packages=nginx` would return all updates with the nginx package and those with the nginx module.